Sources/ResilientDecoding/Resilient.swift

```diff
@@ -47,69 +47,104 @@ public struct Resilient<Value: Decodable>: Decodable {
   func map<T>(transform: (Value) -> T) -> Resilient<T> {
     Resilient<T>(transform(wrappedValue), outcome: outcome)
   }
-  
+
   #if DEBUG
-  /**
+    /**
```

Comment:
I guess Xcode has changed how it handles indenting within `#if` statements since we first wrote this file.